crates/pop-cli/tests/contract.rs

```diff
@@ -80,9 +80,13 @@ async fn contract_lifecycle() -> Result<()> {
 	// Test that all templates are generated correctly
 	generate_all_the_templates(temp_dir)?;
 	// pop new contract test_contract (default)
-	let mut command = pop(temp_dir, ["new", "contract", "test_contract", "--template", "standard"]);
+	let mut command = pop(
+		temp_dir,
+		["new", "contract", "test_contract", "--template", "standard", "--with-frontend=typink"],
+	);
 	assert!(command.spawn()?.wait()?.success());
 	assert!(temp_dir.join("test_contract").exists());
+	assert!(temp_dir.join("test_contract").join("frontend").exists());
```

Comment:
We should test `pop up frontend` here too.

@AlexD10S but that does not return immediately, right? If so it's going to be hard to test 🤔